yarn-project/simulator/src/rollup/rollup.ts

```diff
@@ -146,6 +166,42 @@ export class RealRollupCircuitSimulator implements RollupSimulator {
     return result;
   }
 
+  /**
+   * Simulates the block root rollup circuit from its inputs.
+   * @param input - Inputs to the circuit.
+   * @returns The public inputs as outputs of the simulation.
+   */
+  public async blockRootRollupCircuit(input: BlockRootRollupInputs): Promise<BlockRootOrBlockMergePublicInputs> {
+    const witnessMap = convertBlockRootRollupInputsToWitnessMap(input);
+
+    const witness = await this.wasmSimulator.simulateCircuit(
```

Comment:
Unsure whether the new circuits should use `this.wasmSimulator` or `this.simulationProvider`? Used `wasmSimulator` to match with `merge` for now.

Reply:
Given block root rollup is similar to the old root rollup, and that one used wasm, I'd stick with that one. Eventually we should build a NAPI/FFI interface for the simulator as well as for bb...